lib/block_view.go

```diff
@@ -6367,6 +6571,65 @@ func (bav *UtxoView) ConnectBlock(
 	return utxoOps, nil
 }
 
+func (bav *UtxoView) Preload(bitcloutBlock *MsgBitCloutBlock) error {
```

Comment:
This seems cool-- just to clarify, the idea here is that rather than doing a call to postgres every time we need something in the block that isn't in the view, we try and do a few batch queries up front, correct?

Reply:
Correct!